fastlane/Pluginfile

```diff
@@ -2,7 +2,7 @@
 #
 # Ensure this file is checked in to source control!
 #
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 1.3.1'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 1.4.0'
```

Comment:
Made the same mistake on WPAndroid until I realized a bit later and had to fix it 😅 : we should always stay with only 2 digits (`<major>.<minor>`) when using the permissive operator `~>` so that SemVer works as expected.

With `~> 1.4.0` only the last digit is allowed to change to bigger values so it will catch hypothetical versions `1.4.1` and `1.4.2` but not `1.5.x`; while for semver purposes we _do_ want to get `1.5.x` versions (we just don't want `2.x.y` breaking change versions, at least not automatically). Which is why we need `~> 1.4` instead of `~> 1.4.0` here.

```suggestion
gem 'fastlane-plugin-wpmreleasetoolkit', '~> 1.4'
```

Reply:
Fixed in 7cb2249196a69006b61009fbce779c3955a7f77b.